Client/Frontend/Autofill/CreditCard/CreditCardSettingsView/CreditCardSettingsEmptyView.swift

```diff
@@ -6,39 +6,62 @@ import Foundation
 import SwiftUI
 
 struct CreditCardSettingsEmptyView: View {
+    struct Colors {
+        let titleTextColor: Color
+        let subTextColor: Color
+        let toggleTextColor: Color
+    }
+
+    let colors: Colors
+    @ObservedObject var toggleModel: ToggleModel
+
     var body: some View {
         ZStack {
-            Color(UIColor.clear)
+            UIColor.clear.color
                 .edgesIgnoringSafeArea(.all)
-            VStack {
-                Spacer()
-                Spacer()
-                    .frame(height: 25)
-                Image(ImageIdentifiers.creditCardPlaceholder)
-                    .resizable()
-                    .frame(width: 200, height: 200)
-                    .aspectRatio(contentMode: .fit)
-                    .fixedSize()
-                    .padding([.top], 10)
-                Text(String.CreditCard.Settings.EmptyListTitle)
-                    .font(.system(size: 22))
-                    .padding(.leading, 10)
-                    .padding(.trailing, 10)
-                Text(String.CreditCard.Settings.EmptyListDescription)
-                    .font(.system(size: 16))
-                    .multilineTextAlignment(.center)
-                    .lineLimit(2)
-                    .padding(.leading, 10)
-                    .padding(.trailing, 10)
-                    .padding([.top], -5)
-                Spacer()
+            GeometryReader { proxy in
+                ScrollView {
+                    VStack {
+                        CreditCardAutofillToggle(
+                            textColor: colors.toggleTextColor,
+                            model: toggleModel)
+                        Spacer()
+                        Image("credit_card_placeholder")
```

Comment:
nit: Maybe we can switch it back to `ImageIdentifiers.creditCardPlaceholder` if it's the same